### PR TITLE
fix(Carousel): Add workaround for zero width webviews. Remount when webview size changes from 0 to any other value

### PR DESCRIPTION
**Pull request by pladaria**

### CODE REVIEW

**Review by marcoskolodny — APPROVED** [review]

File: src/index.tsx [hunk 1]
```diff
@@ -114,7 +114,9 @@ export {
     PageBullets,
     CarouselContextProvider,
     useCarouselContext,
-    CarouselContextConsummer,
+    /** @deprecated Contains a typo. Please use CarouselContextConsumer */
```
Comment (atabel) [inline src/index.tsx:117]: :O

**Review by atabel — COMMENTED** [review]
I'd try to push for a native fix. If Android devs can't find a solution, we may need to release this workaround

**Comment by pladaria** [comment]
> I'd try to push for a native fix. If Android devs can't find a solution, we may need to release this workaround

Android ticket: https://jira.tid.es/browse/ANDROID-14348

As commented in teams, merging this while they investigate

**Pull request merged**